### PR TITLE
Add term: Javascript Objects: .isFrozen()

### LINKED ISSUE #4068: [Term Entry] JavaScript Objects .isFrozen()
### Reason/inspiration (optional)

We would like a new term entry on the `Object.isFrozen()` method for [Objects](https://www.codecademy.com/resources/docs/javascript/objects): isFrozen. The entry should go in a new file under `docs/content/javascript/concepts/objects/terms/isFrozen/isFrozen.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```js `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Jamesllllllllll**
### Description

Add term entry for **Javascript Objects: `isFrozen()`**

**NOTE:** I'm not sure if I should be providing any explanation for what **frozen** is in this entry. I started learning about **configurability** and **writability** on the properties, and it seemed like it should belong in its own entry. I welcome any input on this!

Closes #4068 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/objects/terms/isFrozen/isFrozen.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+};
+
+console.log(Object.isFrozen(avgAnnualTemps));
+
+Object.freeze(avgAnnualTemps);
+
+console.log(Object.isFrozen(avgAnnualTemps));
+```
+
+The output would be:
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/isFrozen/isFrozen.md:40]:
```suggestion
The above example will give the following output:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Only one change

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review 🚀

File: content/javascript/concepts/objects/terms/isFrozen/isFrozen.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  - 'Web Development'
+Tags:
+  - 'Methods'
+  - 'Objects'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`.isFrozen()`** static method returns `true` if the given object is _frozen_, and `false` if it is not.
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isFrozen/isFrozen.md:14]:
```suggestion
The **`.isFrozen()`** static method returns `true` if the given object is frozen, and `false` if it is not.
```
Could you write when the object is frozen (Conditions), and add the line in the above statement?

File: content/javascript/concepts/objects/terms/isFrozen/isFrozen.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Syntax
+
+```pseudo
+Object.isFrozen(newObject);
+```
+
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isFrozen/isFrozen.md:21]: Add the description for the `newObject` parameter in one line below the syntax.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@Jamesllllllllll I have mentioned two changes, please make these changes and ping me when you are done.

**Comment by Jamesllllllllll** [comment]
Thanks, @avdhoottt - let me know if I correctly resolved your suggestions.

File: content/javascript/concepts/objects/terms/isFrozen/isFrozen.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+The **`.isFrozen()`** static method returns `true` when the given object is frozen, and `false` if it is not.
+
+## Syntax
+
+```pseudo
+Object.isFrozen(newObject);
+```
+
+`newObj` is the object to be checked for its frozen state.
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isFrozen/isFrozen.md:22]:
```suggestion
- `newObj`: It is the object to be checked for its frozen state.
```

File: content/javascript/concepts/objects/terms/isFrozen/isFrozen.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+```js
+const avgAnnualTemps = {
+  Sydney: '18.7 celcius',
+  Reykjavik: '5.3 celcius',
+  Tokyo: '16.66 celcius',
+};
+
+console.log(Object.isFrozen(avgAnnualTemps));
+
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isFrozen/isFrozen.md:36]:
```suggestion
```

File: content/javascript/concepts/objects/terms/isFrozen/isFrozen.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+```js
+const avgAnnualTemps = {
+  Sydney: '18.7 celcius',
+  Reykjavik: '5.3 celcius',
+  Tokyo: '16.66 celcius',
+};
+
+console.log(Object.isFrozen(avgAnnualTemps));
+Object.freeze(avgAnnualTemps);
+
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isFrozen/isFrozen.md:37]:
```suggestion
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Hey @Jamesllllllllll, The parameter description is fine but you have not mentioned 'what are the conditions in which we call that the object is frozen'. Please add that in the introduction paragraph or as a note below it.

**Comment by Jamesllllllllll** [comment]
> Hey @Jamesllllllllll, The parameter description is fine but you have not mentioned 'what are the conditions in which we call that the object is frozen'. Please add that in the introduction paragraph or as a note below it.

Thanks, I added a description of the conditions of a frozen object. I also **bolded** the first instance of the word frozen per the Docs Style Guide.

File: content/javascript/concepts/objects/terms/isFrozen/isFrozen.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  - 'Web Development'
+Tags:
+  - 'Methods'
+  - 'Objects'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`.isFrozen()`** static method returns `true` when the given object is _frozen_, and `false` if it is not. An object is considered frozen when you cannot add new properties to it, cannot change the existing properties and cannot delete the existing properies. For example, you can freeze an with the`Object.freeze()` static method.
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isFrozen/isFrozen.md:14]:
```suggestion
The **`.isFrozen()`** static method returns `true` when the given object is frozen, and `false` if it is not. An object is considered frozen when you cannot add new properties to it, cannot change the existing properties, and cannot delete the existing properties. For example, you can freeze an with the `Object.freeze()` static method.
```

File: content/javascript/concepts/objects/terms/isFrozen/isFrozen.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+```pseudo
+Object.isFrozen(newObject);
+```
+
+- `newObj`: It is the object to be checked for its frozen state.
+
+## Example
+
+Checking if an object is frozen.
```
Comment (avdhoottt) [inline content/javascript/concepts/objects/terms/isFrozen/isFrozen.md:26]:
```suggestion
The following example checks if an object is frozen.
```

**Review by avdhoottt — APPROVED** [review]
Looks good to be merged.

**Pull request merged**